protos/sigstore_verification.proto

```diff
@@ -162,3 +162,164 @@ message Input {
+        // OID 1.3.6.1.4.1.57264.1.15
+        string sourceRepositoryIdentifier = 15;
+
+        // Source repository owner URL of the owner of the source repository that the build was based on.
+        // OID 1.3.6.1.4.1.57264.1.16
+        string sourceRepositoryOwnerURI = 16;
+
+        // Immutable identifier for the owner of the source repository that the workflow was based upon.
+        // OID 1.3.6.1.4.1.57264.1.17
+        string sourceRepositoryOwnerIdentifier = 17;
+
+        // Build Config URL to the top-level/initiating build instructions.
+        // OID 1.3.6.1.4.1.57264.1.18
+        string buildConfigURI = 18;
+
+        // Immutable reference to the specific version of the top-level/initiating build instructions.
+        // OID 1.3.6.1.4.1.57264.1.19
+        string buildConfigDigest = 19;
+
+        // Event or action that initiated the build.
+        // OID 1.3.6.1.4.1.57264.1.20
+        string buildTrigger = 20;
+
+        // Run Invocation URL to uniquely identify the build execution.
+        // OID 1.3.6.1.4.1.57264.1.21
+        string runInvocationURI = 21;
+
+        // Source repository visibility at the time of signing the certificate.
+        // OID 1.3.6.1.4.1.57264.1.22
+        string sourceRepositoryVisibilityAtSigning = 22;
+}
+
+message SignatureResult {
+        // If the bundle contains a public key, the public_key_id field will
+        // contain the public key ID
+        string public_key_id = 1;
+
+        // If the bundle contains a certificate, the certificate field will
+        // contain metadata about the certificate
+        CertificateSummary certificate = 2;
+}
+
+message TimestampResult {
+        // The type of timestamp verification; may be `Tlog`, `TimestampAuthority`, or `CurrentTime`
+        string type = 1;
+
+        // The URI of the timestamp provider; may contain the base URL of the
+        // TSA or Tlog that was used to establish a timestamp
+        string uri = 2;
+
+        // The actual timestamp time that was verified
+        google.protobuf.Timestamp timestamp = 3;
+}
+
+message UnverifiedTimestampResult {
+        // Data about the timestamp that was unverified
+        TimestampResult result = 1;
+
+        // Reason or error text describing why this timestamp was not verified
+        string reason = 2;
+}
+
+message UnverifiedData {
+        repeated UnverifiedTimestampResult timestamps = 1;
+}
+
+// Result captures the result of the verification
+message Result {
+        // MUST be application/vnd.dev.sigstore.verificationresult+json;version=0.1
+        string media_type = 1;
+
+        // MUST be either "success" or "failure"
+        string status = 2;
+
+        // SHOULD contain an error message if status == "failure"
+        string error = 3;
+
+        // If the bundle contains a DSSE, this result shonuld contain the
+        // deserialized and verified statement
+        string statement = 3;
```

Comment:
bytes instead of string?

Reply:
I think the comments should be rephrased a bit. 
```
// If the bundle contains DSSE wrapped in-toto statement , this result contains the
// deserialized in-toto statement
```

As an in-toto statement is a JSON encoded object, i.e. a string.